sql/2025/fonts/performance/fonts_format_file.sql

```diff
@@ -0,0 +1,44 @@
+-- Section: Performance
+-- Question: Which file formats are used?
+-- Normalization: Requests and fonts
+
+-- INCLUDE https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/{year}/fonts/common.sql
+
+WITH
+requests AS (
+  SELECT
+    client,
+    url,
+    FILE_FORMAT(STRING(summary.ext), STRING(summary.mimeType)) AS format,
```

Comment:
Do we need the `STRING` conversion? Aren't these returned as strings anyway?

Reply:
I think all JSON accessors return a value of type JSON. Hence, I have this conversion everywhere.

Reply:
Ah yes you're right.